### ISSUE TITLE
Incorrect `Unnecessary 'div' tag` error

### ISSUE THREAD

**Issue by shadowgate15** [pugjs/pug-lint#185]
- pug-lint version: 2.6.0
- OS: MacOS Monterey 12.0.1

Currently the following line will fail with an `Unnecessary 'div' tag` error.

`div(data-type="text").class`

If it were written as `(data-type="text").class`, pug will fail to render and throw an error.

## Test

```js
it('should not report necessary div tags with classes after attributes', function () {
  assert.equal(linter.checkString('div(data-type="text").class').length, 0);
});
```

**Comment by lazarljubenovic** [comment]
Isn't this equivalent to `.class(data-type="text")`?
